src/data/set/basic.lean

```diff
@@ -536,17 +536,17 @@ theorem subset_inter {s t r : set α} (rs : r ⊆ s) (rt : r ⊆ t) : r ⊆ s 
 @[simp] theorem subset_inter_iff {s t r : set α} : r ⊆ s ∩ t ↔ r ⊆ s ∧ r ⊆ t :=
 (forall_congr (by exact λ x, imp_and_distrib)).trans forall_and_distrib
 
-theorem subset_iff_inter_eq_left {s t : set α} : s ⊆ t ↔ s ∩ t = s :=
-(ext_iff.trans $ forall_congr $ λ x, and_iff_left_iff_imp).symm
+theorem inter_eq_left_iff_subset {s t : set α} : s ∩ t = s ↔ s ⊆ t :=
+(ext_iff.trans $ forall_congr $ λ x, and_iff_left_iff_imp)
 
-theorem subset_iff_inter_eq_right {s t : set α} : t ⊆ s ↔ s ∩ t = t :=
-(ext_iff.trans $ forall_congr $ λ x, and_iff_right_iff_imp).symm
+theorem inter_eq_right_iff_subset {s t : set α} : s ∩ t = t ↔ t ⊆ s :=
+(ext_iff.trans $ forall_congr $ λ x, and_iff_right_iff_imp)
```

Comment:
nit: parens are now unecessary
```suggestion
ext_iff.trans $ forall_congr $ λ x, and_iff_right_iff_imp
```
but lets leave that to a final cleanup and see if the build passes first.

Reply:
Ok, I will change it in my copy and will not push it right now.  As it is quite likely that there will be further changes, it will just merge by default, once I update later on!

Reply:
I'd recommend not changing it till you've fixed everything else or are done with lean for the day, as changing it will result in a super long build!